# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1300570559 25200
#      Sat Mar 19 14:35:59 2011 -0700
# Node ID 06cebc9919669deff300647160102efdc59cf895
# Parent  33716f289ba59407835d7e2c969ced18ac63c6c7
Add runlength function to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-19  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/stats.txi: Add runlength function to documentation.
+
 2010-03-19  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/plot.txi: Add isocolors, isonormals, isosurface functions
 	to documentation.
 
 2010-03-19  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/basics.txi: Add dump_prefs to documentation.
diff --git a/doc/interpreter/stats.txi b/doc/interpreter/stats.txi
--- a/doc/interpreter/stats.txi
+++ b/doc/interpreter/stats.txi
@@ -122,16 +122,18 @@ different measures from those of the bas
 @DOCSTRING(nchoosek)
 
 @DOCSTRING(perms)
 
 @DOCSTRING(ranks)
 
 @DOCSTRING(run_count)
 
+@DOCSTRING(runlength)
+
 @DOCSTRING(probit)
 
 @DOCSTRING(logit)
 
 @DOCSTRING(cloglog)
 
 @DOCSTRING(mahalanobis)
 
